fmn/rules/utils.py

```diff
@@ -67,6 +72,32 @@ def get_fas(config):
     return _FAS
 
 
+def _paginate_pagure_data(url, params, key):
```

Comment:
The root cause was that pagure doesn't allow a group as main maintainer of a project so the sync script for failing, it was saying so in the logs but the logs are verbose so I missed it until @jeremycline pointed it out.
So I went with: if the PoC is a group, use as main admin the first person in the project who is not a group and we will need to use the override repo to put back in place the group as PoC in BZ.